.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+Resolves #ISSUE.
+
+**No (!) PRs should be merged without at least one review from a different team member of the Mondo Technical team.**
```

Comment:
```suggestion
<!--- **No (!) PRs should be merged without at least one review from a different team member of the Mondo Technical team.** -->
```
Then again, maybe it's good that this isn't a comment so it's more visible.

Reply:
Solved by your other suggestion (adding it to checklist)